gradle/gradle-jdks-functions.sh

```diff
@@ -127,7 +127,6 @@ install_and_setup_jdks() {
     elif [ ! -f "$jdk_installation_directory/bin/java" ]; then
       echo "Java executable not found in $jdk_installation_directory/bin/java, re-installing the JDK...."
     else
-      echo "JDK installation $jdk_installation_directory was already configured"
```

Comment:
Mmm, this is noisy, it was added in #452. We should probably have some test that we don't print anything when the JDK is already installed (to be added later).